### DISCUSSION TITLE
Is it possible to hook it up with apple HomeKit

### DISCUSSION THREAD

**Discussion by alifarooq1991** [revk/xESP32-Faikout#91] [Ideas]
Is it possible to hook it up with apple HomeKit [esp-homekit-devices](https://github.com/RavenSystem/esp-homekit-devices)

**Comment by revk** [comment]
Not quite as simple as it sounds but that is my plan.

The matter stuff is not quite as simple as just adding a managed component yet. I am sort of waiting for that. There looks to be more that one current approach or it.

But yes, plan is matter, which talks HomeKit, is coming to the code some time soon.

**Comment by njh** [comment]
I am currently using HomeKit with Faikin via [Homebridge](https://github.com/homebridge/homebridge) + [homebridge-daikin-local](https://github.com/cbrandlehner/homebridge-daikin-local).

Also been looking at using [MQTT-Thing](https://github.com/arachnetech/homebridge-mqttthing) with Faikin but have been having [trouble getting the status endpoint to work](https://github.com/revk/ESP32-Faikin/issues/74).

But adding native HomeKit support would be super-cool 😀

**Comment by revk** [comment]
I am planning to get back to this soon.

**Comment by revk** [comment]
At this point I am still not able to make matter play with my existing set up (WiFi AP, and web to set wifi settings, and so on). It is close, and looking at the config options it should work, but is giving annoying errors that I can't even find with google. So at this point I end up with a separate build which can do Matter, and once on WiFi you can do things via WiFi/Web pages as well.

This is all testing on a simple LED app. Having given up on a "one-app solution" for now, I'll concentrate on setting it up more sensibly and then integrating with Faikin. It is taking time, sorry.
